### ISSUE TITLE
Not completed?

### ISSUE THREAD

**Issue by Venryx** [DanEdens/GearVr_Plugin#1]
It looks like this plugin was not developed far enough to be working, correct? (I notice that the `GearVr_Plugin` class has nothing but `pass` methods)

**Comment by MadTinker** [comment]
Yeah I started this up really excited, but the peices I that would have needed, stopped working because of various updates. I'll have to take another look tho. The gear vr controller would be so cool if I could map it!

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by Venryx** [comment]
Some of these repositories may be helpful: https://github.com/jsyang/gearvr-controller-webbluetooth/issues/8#issuecomment-739495143
